core/unix/rseq_linux.c

```diff
@@ -161,6 +227,7 @@ d_r_rseq_init(void)
     rseq_cs_table = generic_hash_create(GLOBAL_DCONTEXT, INIT_RSEQ_CS_TABLE_SIZE, 80,
                                         HASHTABLE_SHARED | HASHTABLE_PERSISTENT,
                                         rseq_cs_free _IF_DEBUG("rseq_cs table"));
+    rseq_check_glibc_support();
     /* Enable rseq pre-attach for things like dr_prepopulate_cache(). */
     if (rseq_is_registered_for_current_thread())
         rseq_locate_rseq_regions();
```

Comment:
This will call rseq_check_glibc_support a 2nd time -- do we need both?

Reply:
Removed.